commands/drawCommand.js

```diff
@@ -0,0 +1,123 @@
+import axios from "axios";
+import { MessageAttachment } from "discord.js";
+import db from "../db.js";
+
+export const RARITIES = [
+  "Common",
+  "Rare",
+  "Super Rare",
+  "Holofoil Rare",
+  "Ultra Rare",
+  "Ultimate Rare",
+  "Secret Rare",
+  "Ultra Secret Rare",
+  "Secret Ultra Rare",
+  "Prismatic Secret Rare",
+  "Parallel Rare",
+  "Parallel Common",
+  "Super Parallel Rare",
+  "Ultra Parallel Rare",
+  "Starfoil Rare",
+  "Ghost Rare",
+  "Gold Ultra Rare",
+];
+
+const drawCommand = (message) => {
+  const ref = db.ref("users");
+  ref.once("value", (snapshot) => {
+    const userData = snapshot.val();
+    const userID = message.author.id;
+    const user = userData[userID];
+    const userDoesNotExist = !user;
+    let timeDifference;
+    if (user) {
+      const lastBotUse = user.lastDrawTime;
+      timeDifference = Date.now() - lastBotUse;
+      const userInventory = userData[userID].inventory;
+      const inventorySize = Object.keys(userInventory).length;
+      const DECK_SIZE = 60;
+      if (inventorySize >= DECK_SIZE) {
+        const numOfCardsExceeded = inventorySize - (DECK_SIZE - 1);
+        message.reply(
+          `you have exceeded the deck limit. You must remove ${numOfCardsExceeded} card${
+            numOfCardsExceeded > 1 ? "s" : ""
+          } before you can draw again.`
+        );
+        return;
+      }
+    }
+
+    // 1800000 is 30 minutes in milliseconds
+    if (userDoesNotExist || timeDifference > 60000) {
+      axios
+        .get("https://db.ygoprodeck.com/api/v7/randomcard.php")
+        .then((response) => {
+          let highestRarityIndex = 0;
+          const card = response.data;
+          // eslint-disable-next-line no-plusplus
```

Comment:
try to avoid using `eslint-disable-next-line` unless absolutely necessary -- in this case, it's implicitly telling you that you could rewrite this without a for loop